.gitattributes

```diff
@@ -1,3 +1,4 @@
 *.py	eol=lf
 *.rst	eol=lf
-*.md	eol=lf
\ No newline at end of file
+*.md	eol=lf
+*.mdx   eol=lf
```

Comment:
Needed to add this for Windows because fix-copies and fixup were incorrectly modifying index.mdx and serialization.mdx (in docs/source/en)